[2024-03-15 21:13:07,798] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-15T21:12:00+00:00 [queued]>
[2024-03-15 21:13:07,828] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-15T21:12:00+00:00 [queued]>
[2024-03-15 21:13:07,829] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 21:13:07,829] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-15 21:13:07,829] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-15 21:13:07,843] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_lr_score> on 2024-03-15T21:12:00+00:00
[2024-03-15 21:13:07,855] {standard_task_runner.py:52} INFO - Started process 62057 to run task
[2024-03-15 21:13:07,860] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_lr_score', '2024-03-15T21:12:00+00:00', '--job-id', '3911', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpwdt1jrap', '--error-file', '/tmp/tmp6y4ubhf_']
[2024-03-15 21:13:07,861] {standard_task_runner.py:77} INFO - Job 3911: Subtask compute_lr_score
[2024-03-15 21:13:07,927] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_lr_score 2024-03-15T21:12:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-15 21:13:08,009] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_lr_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-15T21:12:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-15T21:12:00+00:00
[2024-03-15 21:13:08,335] {python.py:151} INFO - Done. Returned value was: None
[2024-03-15 21:13:08,353] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_lr_score, execution_date=20240315T211200, start_date=20240315T211307, end_date=20240315T211308
[2024-03-15 21:13:08,412] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-15 21:13:08,437] {local_task_job.py:151} INFO - Task exited with return code 0
